abletonlink/riscv64.patch

```diff
@@ -0,0 +1,13 @@
+diff --git PKGBUILD PKGBUILD
+index 392f00d1..6afed9d5 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -35,6 +35,8 @@ prepare() {
+ }
+ 
+ build() {
++  export CFLAGS="$CFLAGS -ffp-contract=off"
++  export CXXFLAGS="$CXXFLAGS -ffp-contract=off"
```

Comment:
Can we have more info here? It's not immediately clear to me why `-ffp-contract` should be turned off.